src/contrib/persistence/Akka.Persistence.Sqlite/Journal/BatchingSqliteJournal.cs

```diff
@@ -0,0 +1,119 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="BatchingSqliteJournal.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Configuration;
+using System.Data.Common;
+using System.Data.SQLite;
+using Akka.Configuration;
+using Akka.Persistence.Sql.Common.Journal;
+
+namespace Akka.Persistence.Sqlite.Journal
+{
+    public sealed class BatchingSqliteJournalSetup : BatchingSqlJournalSetup
```

Comment:
Will this automatically get loaded for existing *.SQL plugins that get updated? I.E. do they have to update their HOCON at all to take care of this?